tracer/src/Datadog.Tracer.Native/probes_tracker.h

```diff
@@ -19,17 +19,29 @@ namespace debugger
     private:
         std::recursive_mutex _probeMetadataMapMutex;
         std::unordered_map<shared::WSTRING, std::shared_ptr<ProbeMetadata>> _probeMetadataMap{};
+        // Holds incremental index that is used on the managed side for grabbing an InstrumentedMethodInfo instance (per
+        // instrumented method)
+        inline static std::atomic<int> _nextInstrumentedMethodIndex{0};
+        // Holds incremental index that is used on the managed side for grabbing an InstrumentedProbeInfo instance (per
```

Comment:
```suggestion
        // Holds incremental index that is used on the managed side for grabbing a ProbeData instance (per
```

Reply:
Done in 90e0ba24cfd43eab1fca79e8a887db33e9ddf49e.